pbsmetrics/prometheus/prometheus.go

```diff
@@ -187,6 +188,12 @@ func NewMetrics(cfg config.PrometheusMetrics) *Metrics {
 		"Count of total requests to Prebid Server labeled by account.",
 		[]string{accountLabel})
 
+	metrics.requestsQueueTimer = newHistogram(cfg, metrics.Registry,
+		"request_queue_time",
+		"Seconds request was waiting in queue",
+		[]string{requestTypeLabel, requestStatusLabel},
```

Comment:
There are 5 possible types and 6 possible status. This cardinality results in 30 series and the cost of each series is 14 metrics each. I think 420 new metrics for this timer is a bit much. If we bring the status cardinality down to a bool we get a better 140 new metrics value.

Reply:
Will do if you think we need to merge two error statuses into one. Please see my comment above.

Reply:
Answered above.